### ISSUE TITLE
Integration fails to load with UFH controller

### ISSUE THREAD

**Issue by msp1974** [asantaga/wiserHomeAssistantPlatform#318]
Error in api causes the integration to fail if hub has a UFH controller attached.  Will be fixed in api 0.1.7

```
2022-10-26 13:24:00.380 ERROR (MainThread) [custom_components.wiser.coordinator] Unexpected error fetching wiser (wiser-WiserHeatXXXXXX) data: _WiserUFHController.__init__() takes from 4 to 5 positional arguments but 6 were given
Traceback (most recent call last):
  File "/home/mark/development/core/homeassistant/helpers/update_coordinator.py", line 205, in _async_refresh
    self.data = await self._async_update_data()
  File "/home/mark/development/core/homeassistant/helpers/update_coordinator.py", line 164, in _async_update_data
    return await self.update_method()
  File "/home/mark/development/core/config/custom_components/wiser/coordinator.py", line 134, in async_update_data
    raise ex
  File "/home/mark/development/core/config/custom_components/wiser/coordinator.py", line 115, in async_update_data
    await self.wiserhub.read_hub_data()
  File "/home/mark/development/core/venv/lib/python3.10/site-packages/aioWiserHeatAPI/wiserhub.py", line 159, in read_hub_data
    self._devices = _WiserDeviceCollection(
  File "/home/mark/development/core/venv/lib/python3.10/site-packages/aioWiserHeatAPI/devices.py", line 106, in __init__
    self._build()
  File "/home/mark/development/core/venv/lib/python3.10/site-packages/aioWiserHeatAPI/devices.py", line 162, in _build
    device_class(
TypeError: _WiserUFHController.__init__() takes from 4 to 5 positional arguments but 6 were given
```

**Comment by msp1974** [comment]
@thecem - logged this for your issues

**Comment by thecem** [comment]
Thx, if you need logs let me know.

**Comment by msp1974** [comment]
All good thanks.  I have fixed it but it needs an api release which I will do with next 3.2.1 version.

**Comment by msp1974** [comment]
Have released v3.2.1 with the fix for this issue.  Thanks for raising.

**Comment by thecem** [comment]
Thx, can confirm 3.2.1 is working with the UFH again.
